04/app.js

```diff
@@ -1,5 +1,62 @@
 document.addEventListener('DOMContentLoaded', init);
 
 function init() {
-    console.log('DOM');
-}
\ No newline at end of file
+    console.log('DOMki');
+    getWeatherAtCoordinates();
+}
+
+function getWeatherAtCoordinates() {
+    const formElement = document.querySelector('.form');
+    const apiKey = '519b1b718b664f9c835b63fac827eef9'
+    const fields = formElement.elements;
+    formElement.addEventListener('submit', handleCoordSubmit)
+    
+
+
+    function handleCoordSubmit (e) {
+        e.preventDefault();
+        getServerResponse()
+    }
+
+    function getServerResponse () {
+        const formData = {
+            latitude: fields[0].value,
+            longitude: fields[1].value
+        }
```

Comment:
Lepszym rozwiązaniem (bo można wykorzystać ponownie) jest definiowanie funkcji w zakresie globalnym i przekazywać do niej danych przez parametry. W tym wypadku będzie to:
`function getServerResponse (fields) { ... }`